packages/async_ssl/async_ssl.113.33.06/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "opensource@janestreet.com"
+authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
+homepage: "https://github.com/janestreet/async_ssl"
+bug-reports: "https://github.com/janestreet/async_ssl/issues"
+dev-repo: "https://github.com/janestreet/async_ssl.git"
+license: "Apache-2.0"
+build: [
+  ["./configure" "--prefix" prefix]
+  [make]
+]
+depends: [
+  "ocamlbuild"      {build}
+  "ocamlfind"       {build & >= "1.3.2"}
+  "async"           {>= "113.33.03" & < "113.34.00"}
+  "bin_prot"        {>= "113.33.03" & < "113.34.00"}
+  "conf-openssl"
+  "core"            {>= "113.33.03" & < "113.34.00"}
+  "ctypes"          {>= "0.4.0" & < "0.6.0"}
+  "fieldslib"       {>= "113.33.03" & < "113.34.00"}
+  "js-build-tools"  {build & >= "113.33.04" & < "113.34.00"}
+  "ppx_assert"      {>= "113.33.03" & < "113.34.00"}
+  "ppx_bench"       {>= "113.33.03" & < "113.34.00"}
+  "ppx_driver"      {>= "113.33.03" & < "113.34.00"}
+  "ppx_expect"      {>= "113.33.03" & < "113.34.00"}
+  "ppx_inline_test" {>= "113.33.03" & < "113.34.00"}
+  "ppx_jane"        {>= "113.33.03" & < "113.34.00"}
+  "sexplib"         {>= "113.33.03" & < "113.34.00"}
+  "typerep"         {>= "113.33.03" & < "113.34.00"}
+  "variantslib"     {>= "113.33.03" & < "113.34.00"}
+]
+available: [ ocaml-version = "4.02.3" ]
```

Comment:
This is still only available on 4.02.3 -- so get a constraint failure on 4.03 as before.